packages/mapper/src/mapper.ts

```diff
@@ -78,12 +78,26 @@ export default class Mapper {
     this.messageBus.emit('mongosh:api-call', event);
   }
 
-  use(db): any {
-    if (!(db in this.databases)) {
-      this.databases[db] = new Database(this, db);
+  private _getDatabase(name: string): Database {
+    if (typeof name !== 'string') {
+      throw new MongoshInvalidInputError(
+        `Database name must be a string. Received ${typeof name}.`);
+    }
+
+    if (!name.trim()) {
+      throw new MongoshInvalidInputError('Database name cannot be empty.');
+    }
+
+    if (!(name in this.databases)) {
+      this.databases[name] = new Database(this, name);
```

Comment:
Database names seems to be case insensitive. Should we do something about that here (ie. access ` this.databases[name.toLowerCase()]`), would this cause unexpected behaviours?